test/TESTING_GUIDELINES.md

```diff
@@ -10,6 +10,27 @@ Guide to UI tests for Web SDK.
 - `brew cask install chromedriver`
 - **Note:** if you have chromedriver already installed, make sure it's the latest version: `brew cask upgrade chromedriver`
 
+If you would like to run the tests against other browsers...
+
+Firefox:
+
+- `npm install` should take care of installing Gekodriver (Webdriver for Firefox)
+- **Note:** if you have OSX Catalina installed, you will need to run `npm install -g geckodriver`, this is due to this
+  issue https://github.com/mozilla/geckodriver/issues/1629
+  <br/>
+
+Safari:
+
+- Simply having Safari installed on your machine is enough<br/>
+
+MicroSoft Edge:
+
+1. MicroSoft Edge
+2. Microsoft Edge Driver (WedDriver for Edge)
+
+- **Note:** your Microsoft Edge Driver needs to match the Edge browser version you are running and needs to be installed
+  to your path (i.e. on OSX `/usr/local/bin`)
```

Comment:
This can just be on 1 line
```suggestion
- **Note:** your Microsoft Edge Driver needs to match the Edge browser version you are running and needs to be installed to your path (i.e. on OSX `/usr/local/bin`)
```